src/Elliptic/Systems/ScalarGaussBonnet/Equations.hpp

```diff
@@ -0,0 +1,172 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Elliptic/Systems/ScalarGaussBonnet/Tags.hpp"
+#include "Elliptic/Systems/Xcts/Tags.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/MakeWithValue.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+class DataVector;
+namespace PUP {
+class er;
+}  // namespace PUP
+namespace sgb {
+struct Fluxes;
+struct Sources;
+}  // namespace sgb
+/// \endcond
+
+namespace sgb {
+
+/*!
+ * \brief Compute the fluxes \f$F^i=\gamma^{ij}\partial_j u(x)\f$ for the scalar
+ * equation in sGB gravity on a spatial metric \f$\gamma_{ij}\f$.
+ */
+void curved_fluxes(gsl::not_null<tnsr::I<DataVector, 3>*> flux_for_field,
+                   const tnsr::II<DataVector, 3>& inv_conformal_metric,
+                   const tnsr::I<DataVector, 3>& shift,
+                   const Scalar<DataVector>& lapse,
+                   const Scalar<DataVector>& conformal_factor,
+                   const tnsr::i<DataVector, 3>& field_gradient);
+
+/*!
+ * \brief Compute the fluxes \f$F^i=\gamma^{ij}\partial_j u(x)\f$ for the scalar
+ * equation in sGB gravity on a spatial metric \f$\gamma_{ij}\f$ on a face
+ * normal.
+ */
+void face_fluxes(gsl::not_null<tnsr::I<DataVector, 3>*> flux_for_field,
+                 const tnsr::II<DataVector, 3>& inv_conformal_metric,
+                 const tnsr::I<DataVector, 3>& shift,
+                 const Scalar<DataVector>& lapse,
+                 const Scalar<DataVector>& conformal_factor,
+                 const tnsr::i<DataVector, 3>& face_normal,
+                 const Scalar<DataVector>& field);
+
+/*!
+ * \brief Add the source terms \f$S=-\Gamma^i_{ij}v^j\f$ for the scalar equation
```

Comment:
This also adds the other linear terms. Please add the full latex expression.

Reply:
I've adjusted the comment to be a little more descriptive as well.